draft-ietf-ppm-dap.md

```diff
@@ -666,6 +669,9 @@ requests with HTTP status code 200 OK and an `HpkeConfig` value:
 [TODO: Allow aggregators to return HTTP status code 403 Forbidden in deployments
 that use authentication to avoid leaking information about which tasks exist.]
 
+[OPEN ISSUE: Decide whether aggregators should return 403 or 404 if `max_task_lifetime`
+for the given `task-id` has passed.]
```

Comment:
Let's not add new language referencing HTTP error codes more specific than 400 or 500 since [RFC 9205 cautions against that](https://www.rfc-editor.org/rfc/rfc9205#name-using-http-status-codes). In any case, I don't think we should require that aggregators stop advertising HPKE config for some task ID at any moment in time. The reason is that HPKE configs are likely to be cached for a long time (elsewhere the document explicitly suggests caching them for days), meaning it'd be difficult for implementations to meaningfully guarantee that they'll stop serving a task's HPKE config at the precise instant of task expiration. I don't see any harm in an aggregator continuing to serve expired HPKE configs because the subsequent upload requests to an expired task will reliably fail anyway.